configuration/reports/reportdescriptors/dataexports/sql/j9CaseRegistration.sql

```diff
@@ -5,24 +5,29 @@ select concept_id into @edd from concept where uuid = '3cee56a6-26fe-102b-80cb-0
 select concept_id into @add from concept where uuid = '5599AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 select relationship_type_id into @aParentToB from relationship_type  where uuid = '8d91a210-c2cc-11de-8d13-0010c6dffd0f' ;
 select person_attribute_type_id into @tele from person_attribute_type pat where pat.uuid =  '14d4f066-15f5-102d-96e4-000c29c2a5d7';
+select encounter_type_id into @obgynEncId from encounter_type et where uuid = 'd83e98fd-dc7b-420f-aa3f-36f648b4483d' ;
+select encounter_type_id into @delEncId from encounter_type et  where uuid = '00e5ebb2-90ec-11e8-9eb6-529269fb1459' ;
 -- SET @locale = ifnull(@locale, GLOBAL_PROPERTY_VALUE('default_locale', 'en'));
 
 drop TEMPORARY TABLE IF EXISTS temp_J9_patients;
-
 create temporary table temp_J9_patients
 (
     patient_id int(11),
+    patient_program_id int(11),
     patient_uuid char(38),
     given_name varchar(50),
     family_name varchar(50),
     ZL_identifier varchar(50),
     Dossier_identifier varchar(50),
+    J9_group_max_datetime datetime,
     J9_group text,
     J9_program varchar(50),
     telephone varchar(50),
     birthdate date,
     age int(3),
+    edd_max_datetime datetime,
     estimated_delivery_date date,
+    add_max_datetime datetime,
     actual_delivery_date date,
```

Comment:
@dmdesimone I don't see where the new columns - J9_group_max_datetime, edd_max_datetime, and add_max_datetime are used.  We these added for something you didn't end up needing them for, and then never removed.  Can we remove them to eliminate any future confusion?

Reply:
yeah... exactly.  I went through an iteration of another approach which wasn't any faster.  I will remove these!